### PR TITLE
[REV-1442] Add success log when the script completes successfully

### PR DESCRIPTION
**Pull request by jhan217**
# What did we do?
We added a success log to note when the script completes successfully.

# Why are we doing this?
So we have confirmation that the script completed successfully (regardless of whether the file was imported or downloaded). This will allow us to set the appropriate alerting + monitoring on the script.

# How did we test this?
Updated the unit test for successful download to check for the success logs.

### CODE REVIEW

File: ecommerce/core/management/commands/tests/test_download_sdn_fallback.py [hunk 1]
```diff
@@ -33,7 +33,23 @@ def test_handle_pass(self, mock_response):
         """ Test using mock response from setup, using threshold it will clear"""
 
         mock_response.return_value = self.test_response
-        call_command('populate_sdn_fallback_data_and_metadata', '--threshold=0.0001')
+
+        with LogCapture(self.LOGGER_NAME) as log:
+            call_command('populate_sdn_fallback_data_and_metadata', '--threshold=0.0001')
+
+            log.check(
+                (
```
Comment (jhan217) [inline ecommerce/core/management/commands/tests/test_download_sdn_fallback.py:41]: This log message is technically not part of this PR but was needed for the log checking / part of the success messaging / easy enough to add.

File: ecommerce/core/management/commands/populate_sdn_fallback_data_and_metadata.py [hunk 2]
```diff
@@ -63,6 +63,8 @@ def handle(self, *args, **options):
                         if metadata_entry:
                             logger.info('SDNFallback: IMPORT SUCCESS: Imported SDN CSV. Metadata id %s',
                                         metadata_entry.id)
+
+                        logger.info('SDNFallback: DOWNLOAD SUCCESS: Successfully downloaded the SDN CSV.')
```
Comment (dianekaplan) [inline ecommerce/core/management/commands/populate_sdn_fallback_data_and_metadata.py:67]: This gets the job done, to make a 'DOWNLOAD SUCCESS' log in both cases!  

The only thing I'd note is slightly noisy logs; now both scenarios will log two messages: 
- scenario 1: we import new data- here we log:  
SDNFallback: IMPORT SUCCESS: Imported SDN CSV. Metadata id %s
DOWNLOAD SUCCESS: Successfully downloaded the SDN CSV.
- scenario 2: we skip because data is the same- here we log: 
SDNFallback: The CSV file has not changed, so skipping import. The file_checksum was %s (from models.py)
DOWNLOAD SUCCESS: Successfully downloaded the SDN CSV.

 An alternative approach would be to just update the two existing messages to use a common prefix to use the in splunk query.

**Review by dianekaplan — APPROVED** [review]

File: ecommerce/core/management/commands/populate_sdn_fallback_data_and_metadata.py (hunk 2 above)
Comment (julianajlk) [reply ecommerce/core/management/commands/populate_sdn_fallback_data_and_metadata.py:67]: Wondering the same, maybe adding Success to the "skipping import" log (although is that considered a success?), or as Diane mentioned, we can have a common prefix to all

Comment (jhan217) [reply ecommerce/core/management/commands/populate_sdn_fallback_data_and_metadata.py:67]: @MatthewPiatetsky any thoughts on the log messages? If I understand correctly, it sounds like the suggestion is to update the `SDNFallback: The CSV file has not changed..` to `SDNFallback: DOWNLOAD SUCCESS: The CSV file has not changed`

IIRC the `SDNFallback` is the common prefix for all of the SDNFallback logs. There's a slight difference between Download and Import, so that would make sense to keep. The SUCCESS/FAILURE in the logs seem to be done only at the highest level, ie in the command file. 

I'm happy to keep things "noisy" since each log message seems to serve a specific purpose, or change things if there is a common consensus.

**Review by julianajlk — APPROVED** [review]

**Comment by jhan217** [comment]
Confirmed with Matthew over Slack that the doesn't have an opinion. Given the approvals, I'm going to keep the logs as they are now. We can change the wording later if needed; the important piece is that we have logging where we want it.
